scripts/rust_integration_tests.sh

```diff
@@ -0,0 +1,33 @@
+#! /bin/sh
+
```

Comment:
we need to put `set -eu` at the beginning of all scripts.

Reply:
:+1: